doc/api/fs.md

```diff
@@ -230,6 +230,101 @@ try {
 }
 ```
 
+#### `filehandle.createReadStream([options])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `options` {Object}
+  * `encoding` {string} **Default:** `null`
+  * `autoClose` {boolean} **Default:** `true`
+  * `emitClose` {boolean} **Default:** `true`
+  * `start` {integer}
+  * `end` {integer} **Default:** `Infinity`
+  * `highWaterMark` {integer} **Default:** `64 * 1024`
+* Returns: {fs.ReadStream} See [Readable Stream][].
```

Comment:
The `{fs.ReadStream}` and the `[Readable Stream][]` links are going to be redundant here...

Likewise with the `fs.WriteStream` and `[Writable Stream][]` further down.

Reply:
Copied from [`fs.createReadStream`](https://nodejs.org/api/fs.html#fs_fs_createreadstream_path_options) docs: https://github.com/nodejs/node/blob/9fdb7262a766eaee022678e10c0129aee2c8f853/doc/api/fs.md#L1980